.gitignore

```diff
@@ -1,3 +1,6 @@
 vendor/*
 config.yml
 composer.phar
+
+# Library don't need composer.lock on repo
+composer.lock
```

Comment:
There are no need to add `composer.lock` for libraries due to best practices.
http://stackoverflow.com/a/24247443/715329